src/anyio/to_process.py

```diff
@@ -43,7 +43,7 @@ async def run_sync(
     :return: an awaitable that yields the return value of the function.
 
     """
-    async def send_raw_command(pickled_cmd: bytes):
+    async def send_raw_command(pickled_cmd: bytes) -> T_Retval:
```

Comment:
`T_Retval` seems wrong here. It has nothing to do with the final return value of the enclosing function.

Reply:
Ah woops, it's only the last use of send_raw_command that does return a T_Retval